external/llvm-project/mlir/lib/CAPI/IR/IR.cpp

```diff
@@ -734,7 +734,7 @@ static mlir::WalkResult unwrap(MlirWalkResult result) {
   case MlirWalkResultInterrupt:
     return mlir::WalkResult::interrupt();
 
-  case MlirWalkResultSkip:
+  default:
```

Comment:
why was this changed to Default ?

Reply:
The compiler warns that some values are not covered in the switch statement.